src/components/shared/forms/styles/tooltip.scss

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+.tooltip-container {
+  @include rem-font-size(12px);
```

Comment:
The pattern is that `@includes`  should go at the end of the list of properties

Reply:
Hmmm, not sure how I got it in my head they were supposed to be first.